velox/docs/functions/spark/string.rst

```diff
@@ -427,3 +427,26 @@ String Functions
     Returns string with all characters changed to uppercase. ::
 
         SELECT upper('SparkSql'); -- SPARKSQL
+
```

Comment:
The functions in this file and listed in alphabetical order. Please add this function in its correct position.